src/app/_helpers/cache.spec.ts

```diff
@@ -38,11 +38,11 @@ describe('single cache', () => {
   });
 
   it('should not cache an error', () => {
-    const fn = jasmine.createSpy().and.callFake(() => throwError('wrong'));
+    const fn = jasmine.createSpy().and.callFake(() => throwError(new Error('wrong')));
     const cache = new SingleCache<number>(fn);
-    expect(gatherError(cache.get())).toEqual('wrong');
-    expect(gatherError(cache.get())).toEqual('wrong');
-    expect(gatherError(cache.get())).toEqual('wrong');
+    new Array(3).fill(null).map(() => {
+      expect(gatherError(cache.get())).not.toEqual('wrong');
+    });
```

Comment:
My sonarlint complains about mapping as the result is not used. It suggests using `forEach`. 
Why not just do some easy-to-read for loop? Perhaps:
```
    new Array(3).forEach(() => {
      expect(gatherError(cache.get())).not.toEqual('wrong');
    });
```
Or even
```
[1,2,3].forEach(() => {
  expect(gatherError(cache.get())).not.toEqual('wrong');
}
```